pretix_wallet/views.py

```diff
@@ -115,3 +108,11 @@ def create(self, request, *args, **kwargs):
         serializer.is_valid(raise_exception=True)
         self.perform_create(serializer)
         return Response(WalletSerializer(self.wallet).data, status=HTTP_201_CREATED)
+
+
+class WalletRequiredRedirectView(CustomerRequiredMixin, WalletRequiredMixin, View):
```

Comment:
why do we need this?

Reply:
I'll keep this until we've verified the checkout step actually works. If that's the case, we can remove it.